pkg/client/vcwallet/client.go

```diff
@@ -260,9 +307,26 @@ func (c *Client) Query(query *QueryParams) ([]json.RawMessage, error) {
 //		- A verifiable credential with or without proof
 //		- Proof options
 //
-func (c *Client) Issue(credential json.RawMessage, options *ProofOptions) (json.RawMessage, error) {
-	// TODO to be added #2433
-	return nil, fmt.Errorf("to be implemented")
+func (c *Client) Issue(authToken string, credential json.RawMessage,
```

Comment:
I'm curious about the authToken as the first argument pattern. Couldn't Open return a Wallet object containing the authToken?

(and these functions would be on the Wallet object).

@fqutishat @sudeshrshetty @rolsonquadras

Reply:
Will be a nice option if we are only going to use SDK binding.
Since wallet server (REST command controller) is going to be stateless it will have to get new client instance each time, and opening wallet (instantiating kms, which is expensive indeed) each time we create new client instance is not going to be feasible here.